src/constants/serverConnexion.tsx

```diff
@@ -0,0 +1 @@
+export const URL_SERVER = "http://hapi.fhir.org/baseR4/";
```

Comment:
I would advise you to put all your constants in the same file: constants/index.ts

Reply:
Done